Lib/gftools/builder/operations/autohintOTF.py

```diff
@@ -3,4 +3,4 @@
 
 class AutohintOTF(OperationBase):
     description = "Run otfautohint"
-    rule = "otfautohint $args -o $out $in \|\| otfautohint $args -o $out $in --no-zones-stems"
+    rule = "otfautohint $args -o $out $in \\|\\| otfautohint $args -o $out $in --no-zones-stems"
```

Comment:
@arrowtype's traceback shows,

```
otfautohint: error: argument -o/--output: /fonts/Familyname/|| is not a valid path to write to.
```

The or operator looks well escaped. The issue is that the or operator is attached to the filename. Perhaps quoting the $out and $in args will work e.g

```
'otfautohint $args -o "$out" "$in" \|\| otfautohint $args -o "$out" "$in" --no-zones-stems'
```

Reply:
You are right, probably the `$in` and `$out` arguments should be escaped with quotes. I will adjust that. The thing I encountered was actually a Python warning:
![image](https://github.com/user-attachments/assets/10343735-fb79-4557-9aae-cf6bd285ade2)